### PR TITLE
Customize jenkins-operator helmchart with adding 'nodeSelector'

### PR DESCRIPTION
**Pull request by ywang-psee**
# Changes

add 'nodeSelector' item into jenkins-operator helmchart.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes tests (if functionality changed/added)
- [ ] Includes docs (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md#commit-messages)

_See [the contribution guide](https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md) for more details._

## Reviewer Notes
If API changes are included, additive changes must be approved by at least two [OWNERS](https://github.com/jenkinsci/kubernetes-operator/blob/master/OWNERS) and backwards incompatible changes must be approved by [more than 50% of the OWNERS](https://github.com/jenkinsci/kubernetes-operator/blob/master/OWNERS).

### CODE REVIEW

**Comment by ywang-psee** [comment]
@Sig00rd @prryb

**Review by Sig00rd — APPROVED** [review]
lgtm

File: chart/jenkins-operator/templates/jenkins.yaml [hunk 1]
```diff
@@ -85,6 +85,9 @@ spec:
   {{- with .Values.jenkins.notifications }}
   notifications: {{ toYaml . | nindent 4 }}
   {{- end }}
+  {{- with .Values.jenkins.nodeSelector }}
+  nodeSelector: {{ toYaml . | nindent 6 }}
```
Comment (prryb) [inline chart/jenkins-operator/templates/jenkins.yaml:89]:
```suggestion
  nodeSelector: {{ toYaml . | nindent 4 }}
```

**Review by prryb — COMMENTED** [review]
I believe `nindent 4` should be enough.

File: chart/jenkins-operator/templates/jenkins.yaml (hunk 1 above)
Comment (ywang-psee) [reply chart/jenkins-operator/templates/jenkins.yaml]: Hi @prryb , thanks for the noticed, changed in commit 4b75d27bc0f92c1a7f22a751ac43df4ebe5f9fe3

**Comment by ywang-psee** [comment]
Hi, can you please approval this pull request if the change is ok for you please? @prryb. Thanks!

**Review by prryb — APPROVED** [review]

**Comment by cbernard-psee** [comment]
Hi @prryb , do you know the process to see this PR merged into the main branch?

**Comment by ccwumao** [comment]
Hi, can you please give me the write access to merge the pull request, or can you please merged this pr in my place? thanks! @prryb @Sig00rd

**Pull request merged**